### PR TITLE
Added orientation and click-to-toggle classes to FAB

### PR DESCRIPTION
**Pull request by reesmanp**
Issue #157

### CODE REVIEW

File: src/Button.js [hunk 1]
```diff
@@ -112,11 +115,16 @@ Button.propTypes = {
    */
   onClick: PropTypes.func,
   tooltip: PropTypes.string,
-  waves: PropTypes.oneOf(constants.WAVES)
+  waves: PropTypes.oneOf(constants.WAVES),
+  /**
+   * Click-Only
```
Comment (alextrastero) [inline src/Button.js]: Maybe we could have a bit more description here as this is what will show on the documentation

File: src/Button.js [hunk 2]
```diff
@@ -112,11 +115,16 @@ Button.propTypes = {
...
+   */
+  clickOnly: PropTypes.bool
 };
 
 Button.defaultProps = {
-  node: 'button'
+  node: 'button',
+  clickOnly: false
```
Comment (alextrastero) [inline src/Button.js]: I'm not sure we need a default prop for this guy, if he's not supplied the effect will be the same.

Comment (reesmanp) [inline src/Button.js]: Fair enough. I was just being explicit with the default. I will remove this in my next update.

File: src/Button.js (hunk 1 above)
Comment (reesmanp) [inline src/Button.js]: Will do.

**Pull request merged**